app/script_variant.py

```diff
@@ -122,7 +166,9 @@ def romanize_word(word):
 
     romanized_words = []
     if has_kanji(word):
-        romanized_words = romanize_japanese_name_by_name_dict(word)
+        # Add for_index parameter to normalize_word
```

Comment:
I mean, you should add for_index parameter to normalize_word (not romanize_japanese_name_by_name_dict).